tonic/src/transport/mod.rs

```diff
@@ -38,29 +39,31 @@
 //!     .connect()
 //!     .await?;
 //!
-//! channel.call(Request::new(BoxBody::empty())).await?;
+//! channel.call(Request::new(empty_body())).await?;
```

Comment:
nit: should we just fully qualify this instead of hiding the import. Feels a bit confusing as to where this `empty_body` fn comes from.